test/Python/python_runtime.swift

```diff
@@ -6,17 +6,48 @@
 import Python
 import StdlibUnittest
 
+/// The gc module is an interface to the Python garbage collector.
+let gc = Python.import("gc")
+/// The tracemalloc module is a debug tool to trace memory blocks allocated by
+/// Python. It is optionally imported because it is available only in Python 3.
+let tracemalloc = try? Python.attemptImport("tracemalloc")
+
+extension TestSuite {
+  /// Check that running `body` does not cause Python memory leaks.
+  func testWithLeakCheck(_ name: String, body: @escaping () -> Void) {
```

Comment:
```suggestion
  func testWithLeakChecking(_ name: String, body: @escaping () -> Void) {
```

Sorry, I originally suggested `testWithLeakCheck`, then updated my comment to suggest `testWithLeakChecking`.